### ISSUE TITLE
About MobileNet train

### ISSUE THREAD

**Issue by allenxcp** [deepinsight/insightface#129]
@nttstar  I train the MobilNet,But don't got the best reuslt like your paper.
the params is:
**gpu num: 4**
**num_layers 1**
image_size [112, 112]
num_classes 85164
Called with argument: Namespace(batch_size=**512**, beta=1000.0, beta_freeze=0, beta_min=5.0, ckpt=1, ctx_num=4, cutoff=0, data_dir='insightface/datasets/faces_ms1m_112x112', easy_margin=0, emb_size=512, end_epoch=100000, gamma=0.12, image_channel=3, image_h=112, image_w=112, **loss_type=4**, lr=0.1, lr_steps='', margin=4, margin_a=1.0, margin_b=0.0, **margin_m=0.5,** margin_s=64.0, max_steps=0, mom=0.9, network='m1', num_classes=85164, num_layers=1, per_batch_size=128, power=1.0, prefix='../model-m1-insight/model', pretrained='', rand_mirror=1, rescale_threshold=0, scale=0.9993, target='lfw,cfp_fp,agedb_30', use_deformable=0, verbose=2000, version_act='prelu', version_input=1, version_output='E', version_se=0, version_unit=3, wd=0.0005)
init mobilenet 1
**(1, 'E', 3)**

  header = header._replace(label=np.fromstring(s, np.float32, header.flag))
header0 label [3804847. 3890011.]
id2range 85164
3804846
rand_mirror 1

the best result:
lr-batch-epoch: 0.0001 4156 53
testing verification..
(12000, 512)
infer time 18.578805
[lfw][398000]XNorm: 22.040030
[lfw][398000]Accuracy-Flip: **0.99433+-0.00281**
testing verification..
(14000, 512)
infer time 19.498474
[cfp_fp][398000]XNorm: 18.207268
[cfp_fp][398000]Accuracy-Flip: 0.89029+-0.01814
testing verification..
(12000, 512)
infer time 16.765434
[agedb_30][398000]XNorm: 22.209527
[agedb_30][398000]Accuracy-Flip: 0.96267+-0.00889

can you give me some advise?
